test/node/reducers/composer-state-test.js

```diff
@@ -86,8 +86,8 @@ test('#composerStateReducer - addDataToMapComposed: mapState should not be cente
   // init kepler.gl root and instance
   const state = keplerGlReducer({}, registerEntry({id: 'test'})).test;
   const mapStateProperties = {
-    latitude: 33.88608913680742,
-    longitude: -84.43459130456425
+    latitude: 29.23, //33.88608913680742,
```

Comment:
is this a test case? if not put these constant values inside a configuration file

Reply:
yes, It is a test case.